resources/js/views/settings/roles/View.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <div>
+    <h3>
+      {{ id === 'new' ? $t('settings.roles.new') : (
+        viewOnly ? $t('settings.roles.view', { name: $te(`app.roles.${model.name}`) ? $t(`app.roles.${model.name}`) : model.name })
+          : $t('settings.roles.edit', { name: $te(`app.roles.${model.name}`) ? $t(`app.roles.${model.name}`) : model.name })
+      ) }}
+    </h3>
+    <hr>
+
+    <b-form @submit='saveRole'>
+      <b-container fluid>
+        <b-row class='my-1'>
+          <b-col sm='3'>
+            <label for='name'>{{ $t('settings.roles.name') }}</label>
+          </b-col>
+          <b-col sm='9'>
+            <b-form-input :state='errors.name ? false : null' id='name' type='text' v-model='model.name' :disabled='isBusy || viewOnly'></b-form-input>
```

Comment:
Please use these functions or implement them globaly, as we need this again and again: https://github.com/THM-Health/PILOS/blob/master/resources/js/components/Room/SettingsComponent.vue#L326